src/demo/demoUtils.js

```diff
@@ -30,6 +30,7 @@ export const getInitSdkOptions = () => {
       type:'document',
       options: {
         useLiveDocumentCapture: queryParamToValueString.useLiveDocumentCapture === 'true',
+        disableFallback: queryParamToValueString.disableFallback === 'true',
```

Comment:
Also, wouldn't it be more beneficial to be able to determine if a device is hybrid and if so, disable the fallback, without needing to pass a configuration option? This would remove the integration overhead too.

Reply:
We can definitely look for a few key things to try to determine if the device is a hybrid (or Surface like) and disable the fallback based on that. I was thinking it would be more reusable as a configuration, instead of doing a one off for this specific case. @seewah which way would you prefer?

Reply:
Would we want to automatically disable the fallback if an end-user happens to be using a Surface/hybrid though?

Reply:
I discussed this with Afiya. We came to the conclusion that for the purposes of flexibility for future client requests, that it would be good to have this done as a config.  If there is a case where a client will want to use live document capture on a Surface, but wants to be able to fallback to cross device flow or document upload, this will allow us not to have the Surface restricted to no fallbacks. Also, as Barclays tests, if they change their mind about the fallback options, it would be easier to re-enable through the config rather than changing the logic again. I've changed the name to `disableAllFallbacks` to distinguish it from `uploadFallback` more clearly.

Reply:
Sorry i did not see this earlier, as I don't have github notification! I understand the rationale behind this. I am going to have a quick chat with @stefaniacardenas tomorrow and leave further comment, if i have any. Regardless we will close off this conversation tomorrow!